prod/westeurope/internal/api/functions_assets_r3/subnet/terragrunt.hcl

```diff
@@ -0,0 +1,35 @@
+# Common
+dependency "virtual_network" {
+  config_path = "../../../../common/virtual_network"
+}
+
+# Include all settings from the root terragrunt.hcl file
+include {
+  path = find_in_parent_folders()
+}
+
+terraform {
+  source = "git::git@github.com:pagopa/io-infrastructure-modules-new.git//azurerm_subnet?ref=v2.1.11"
+}
+
+inputs = {
+  name = "fn3assets"
+
+  resource_group_name  = dependency.virtual_network.outputs.resource_group_name
+  virtual_network_name = dependency.virtual_network.outputs.resource_name
+  address_prefix       = "10.0.111.0/24"
+
+  delegation = {
+    name = "default"
+
+    service_delegation = {
+      name    = "Microsoft.Web/serverFarms"
+      actions = ["Microsoft.Network/virtualNetworks/subnets/action"]
+    }
+  }
+
+  service_endpoints = [
+    "Microsoft.Web",
+    "Microsoft.AzureCosmosDB",
+  ]
+}
```

Comment:
Could this be reached by `assets.cdn.io.italia.it` ?

Reply:
@AleDore do you mean the function? Yes, it's an origin of the CDN. The function has no restrictions ... anyone that knows the url and the app key can call the function: the cdn passes the app key as an http header.
Am I wrong? Do you see any possible issue here?

Reply:
No It's fine :)